sklearn/partial_dependence.py

```diff
@@ -210,6 +207,12 @@ def partial_dependence(est, target_variables, grid=None, X=None, output=None,
     >>> partial_dependence(gb, [0], **kwargs) # doctest: +SKIP
     (array([[-4.52...,  4.52...]]), [array([ 0.,  1.])])
     """
+    # TODO: The pattern below required to avoid a namespace collision.
+    # TODO: Move below imports to module level import at 0.22 release.
+    from .ensemble._gradient_boosting import _partial_dependence_tree
```

Comment:
@jnothman @amueller can you give a hint on if there's a better way to do this deprecation? Right now the tests are passing locally and online... But if I try to import locally `ensemble`'s `__init__` import of `partial_dependence` (the old one) causes it to fail locally.